stdlib/public/core/LazyCollection.swift

```diff
@@ -10,23 +10,23 @@
 //
 //===----------------------------------------------------------------------===//
 
-public protocol LazyCollectionProtocol: Collection, LazySequenceProtocol 
+public protocol LazyCollectionProtocol: Collection, LazySequenceProtocol
 where Elements : Collection {	}
 
-extension LazyCollectionProtocol {		
-   // Lazy things are already lazy		
-   @inlinable // protocol-only		
-   public var lazy: LazyCollection<Elements> {		
-     return elements.lazy		
-   }		
- }		
-		
- extension LazyCollectionProtocol where Elements: LazyCollectionProtocol {		
-   // Lazy things are already lazy		
-   @inlinable // protocol-only		
-   public var lazy: Elements {		
-     return elements		
-   }		
```

Comment:
Try to keep unrelated whitespace changes out of patches.

Reply:
Should be restored.